src/main/java/io/fusionauth/scim/utils/SCIMPatchTools.java

```diff
@@ -191,8 +194,9 @@ public static ArrayNode convertSCIMPatchToJSONPatch(ObjectMapper objectMapper, J
               // - Add a new op to the result for each matching node. It is plausible we'll match more than one node.
               ObjectNode copy = operation.deepCopy();
               copy.set("path", TextNode.valueOf(path.asText() + "/" + i));
-              copy.remove("value");
-              result2.add(copy);
+              // Temporarily set the value for later sorting and add to list
+              copy.set("value", objectMapper.createArrayNode().add(TextNode.valueOf(path.asText())).add(IntNode.valueOf(i)));
```

Comment:
I chose to temporarily place `path` and `index` in the `value` field in order to avoid having to parse the string for sorting.